src/features/Details/components/HeroHeadline/HeroHeadline.tsx

```diff
@@ -0,0 +1,31 @@
+import cx from 'classnames';
+
+import { Superhero } from '@interfaces/superhero.interface';
+
+import styles from './HeroHeadline.module.scss';
+import { jestIdsMap } from '@utils/jestHelpers';
+
+interface HeroHeadlineProps {
+  name: Superhero['name'];
+  alignment: Superhero['biography']['alignment'];
```

Comment:
**thought** This "reusing" of types is a very interesting approach 

**question** Why didn't you create separate interfaces/types for them?

Reply:
> Why didn't you create separate interfaces/types for them?

@kamil-borecki-cc True, honestly I just didn't think of that.